apps/dashboard/src/components/workflow-editor/steps/push/push-editor.tsx

```diff
@@ -1,28 +1,30 @@
-import { getComponentByType } from '@/components/workflow-editor/steps/component-utils';
-import { PushTabsSection } from '@/components/workflow-editor/steps/push/push-tabs-section';
-import { type UiSchema } from '@novu/shared';
 import { RiCellphoneFill } from 'react-icons/ri';
+import { type UiSchema } from '@novu/shared';
+
+import { getComponentByType } from '@/components/workflow-editor/steps/component-utils';
+import { TabsSection } from '@/components/workflow-editor/steps/tabs-section';
 
 const subjectKey = 'subject';
 const bodyKey = 'body';
 
 type PushEditorProps = { uiSchema: UiSchema };
+
 export const PushEditor = (props: PushEditorProps) => {
   const { uiSchema } = props;
   const { [bodyKey]: body, [subjectKey]: subject } = uiSchema?.properties ?? {};
```

Comment:
I know its not part of this PR just noticed here. 
```suggestion
  const { body, subject } = uiSchema.properties ?? {};
```